financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkstepupverification/LinkStepUpVerificationViewModel.kt

```diff
@@ -73,36 +73,6 @@ internal class LinkStepUpVerificationViewModel @AssistedInject constructor(
         )
     }
 
-    private suspend fun lookupAndStartVerification() = runCatching {
-        getOrFetchSync().manifest.also { requireNotNull(it.accountholderCustomerEmailAddress) }
-    }
-        .onFailure { setState { copy(payload = Fail(it)) } }
-        .onSuccess { manifest ->
-            setState { copy(payload = Loading()) }
-            lookupConsumerAndStartVerification(
-                email = requireNotNull(manifest.accountholderCustomerEmailAddress),
-                businessName = manifest.businessName,
-                verificationType = VerificationType.EMAIL,
-                onConsumerNotFound = {
-                    eventTracker.track(VerificationStepUpError(PANE, ConsumerNotFoundError))
-                    navigationManager.tryNavigateTo(InstitutionPicker(referrer = PANE))
-                },
-                onLookupError = { error ->
-                    eventTracker.track(VerificationStepUpError(PANE, LookupConsumerSession))
-                    setState { copy(payload = Fail(error)) }
-                },
-                onStartVerification = { /* no-op */ },
-                onVerificationStarted = { consumerSession ->
-                    val payload = buildPayload(consumerSession)
-                    setState { copy(payload = Success(payload)) }
-                },
-                onStartVerificationError = { error ->
-                    eventTracker.track(VerificationStepUpError(PANE, StartVerificationError))
-                    setState { copy(payload = Fail(error)) }
-                }
-            )
-        }
```

Comment:
since we can clean up all the return points, now we can use the regular `suspend {} execute` and delegate errors to `onAsync` callbacks.